### PR TITLE
Sharding

### PR DESCRIPTION
**Pull request by USERSATOSHI**

### CODE REVIEW

**Comment by Leref** [comment]
It missing the basic 3 syntax for AoiEngine.
`title`, `description`, `id`.

As well the `intents` method aren't updated.

File: src/guides/sharding.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# Sharding
```
Comment (Leref) [inline src/guides/sharding.md:1]: Not needed

File: src/guides/sharding.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+```
+
+in `bot.js`
+
+```js
+const { Bot } = require("aoi.js");
+const bot = new Bot({
+    token: "DISCORD BOT TOKEN",
+    prefix: "DISCORD BOT PREFIX",
+    intents: ["guilds", "guildMessages"],
```
Comment (Leref) [inline src/guides/sharding.md:78]: Update to proper intents.

**Review by Leref — CHANGES_REQUESTED** [review]

File: src/guides/sharding.md (hunk 2 above)
Comment (USERSATOSHI) [reply src/guides/sharding.md]: done

**Review by Leref — APPROVED** [review]

**Pull request merged**